sql/core/src/test/resources/sql-tests/inputs/array.sql

```diff
@@ -106,3 +106,10 @@ select elt(2, '123', null);
 
 select array(1, 2, 3)[5];
 select array(1, 2, 3)[-1];
+
+-- array_size
+select array_size(array()) = size(array());
```

Comment:
let's test `array_size` only instead of comparing with `size`. We should just test `select array_size(array());`

Reply:
Sounds good!